spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/export/datadog/DatadogPropertiesConfigAdapterTests.java

```diff
@@ -27,14 +27,6 @@
  */
 public class DatadogPropertiesConfigAdapterTests {
 
-	@Test
-	public void apiKeyInferUri() {
```

Comment:
I am not comfortable removing this test. What is the replacement? Can we please have a test that exercises the replacement?